lib/dockerregistry/types.go

```diff
@@ -294,3 +352,21 @@ func (riid RegInvImageDigest) ToRegInvImageTag() RegInvImageTag {
 	}
 	return riit
 }
+
+// ToRegInvImageDigest converts a RegInvImageTag to a RegInvImageDigest.
+func (riit RegInvImageTag) ToRegInvImageDigest() RegInvImageDigest {
+	riid := make(RegInvImageDigest)
+	for imageTag, digest := range riit {
+		id := ImageDigest{}
+		id.ImageName = imageTag.ImageName
+		id.Digest = digest
```

Comment:
I agree, but I think we should postpone this until after this PR --- because we can do it in the other conversion functions in this file. Let's have a refactor/cleanup PR that does all of that in one go.